docs/src/usage/distributed.rst

```diff
@@ -131,7 +131,8 @@ have to :func:`mlx.utils.tree_map` the gradients with following function.
 .. code:: python
 
     def all_avg(x):
-        return mx.distributed.all_sum(x) / mx.distributed.init().size()
+        N = mx.distributed.init().size()
```

Comment:
Explicitly execute the initialization first, even if `mx.distributed.all_sum(x)` is lazily evaluated.

Reply:
I don't think this is necessary.

Reply:
The [Undo aesthetic code changes](https://github.com/danilopeixoto/mlx/commit/280e38e5cca20ee04f3299b1b21cc623cef9e052) commit reversed that. Thanks!